package/MDAnalysis/coordinates/PDB.py

```diff
@@ -631,7 +656,31 @@ def _write_pdb_title(self):
                        "".format(self.start, self.remarks))
 
     def _write_pdb_header(self):
-        if self.first_frame_done == True:
+        """
+        Write PDB header.
+
+        The HEADER_ record is set to :code: `trajectory.header`.
+        The TITLE record explicitly mentions MDAnalysis and contains
+        information about trajectory frame(s).
+        The COMPND record is set to :code:`trajectory.compound`.
+        The REMARKS records are set to :code:`u.trajectory.remarks`
+        The CRYST1_ record specifies the unit cell. This record is set to
+        unitary values (cubic box with sides of 1Å) if unit cell dimensions
```

Comment:
1 Å

Reply:
These docs are good but never visible to users in the online docs. Put anything (such as the CRYST1 stuff) into the class docs.